meta-facebook/yv35-hd/src/platform/plat_class.c

```diff
@@ -269,7 +279,34 @@ void init_platform_config()
 				printf("Unknown the 1OU card type, the voltage of ADC channel-6 is %fV\n",
 				       voltage);
 			} else if (_1ou_status.card_type == TYPE_1OU_RAINBOW_FALLS) {
-				gpio_set(HSC_OCP_GPIO1_R, GPIO_HIGH);
+				if (hsc_module == HSC_MODULE_ADM1278) {
+					gpio_set(HSC_OCP_GPIO1_R, GPIO_HIGH);
+				} else if (hsc_module == HSC_MODULE_LTC4282) {
```

Comment:
Please add else statement to catch invalid hsc_module values.

Reply:
An else statement will be added to handle all conditions.